### ISSUE TITLE
Travis-build fails

### ISSUE THREAD

**Issue by nmueller18** [ISAAKiel/mortAAR#56]
@nevrome: Could you please test the latest master? Locally, everything tests fine, but the travis-build fails. The log says:

```
-- FAILURE (test_life_table.R:8:3): life.table produces the right output with sp
Snapshot of `life.table(nitra_prep, option_spline = 10)` has changed:
`old$x` is an S3 object of class <factor>
`new$x` is a character vector ('0--4', '5--9', '10--14', '15--19', '20--24', ...)

Run `snapshot_accept('life_table')` if this is a deliberate change

â•â• testthat results  â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•â•
FAILURE (test_life_table.R:8:3): life.table produces the right output with spline-function

[ FAIL 1 | WARN 0 | SKIP 0 | PASS 118 ]
Error: Test failures
```

I suppose it has to do with the serialization-option instead of 'json2' but otherwise the test fails, probably because of rounding issues.
Any ideas?

**Comment by nevrome** [comment]
I guess this is caused by your old R version, which does not have `stringsAsFactors = FALSE` as a default when constructing data.frames. The new version is the correct one, but the fact that the old one was generated somewhere (I guess on your computer) means that there is still a call to `cbind` or `data.frame` without `stringsAsFactors = FALSE`. But I can't find it.

**Comment by nmueller18** [comment]
I think I have found it. The old nitra.rda, which I have added later than the rest of the example data, was formatted as `factors` instead of `char`. I have changed this and hope the best. In doing so, I noticed that the command `devtools::use_data` seems to be deprecated (https://stackoverflow.com/questions/54419777/devtoolsuse-data-internal-true). Instead, the command is now supposed to be `usethis::use_data`. Do we have to reference `usethis`, then?
